src/App.tsx

```diff
@@ -1,24 +1,200 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect, useMemo } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { MainFilter } from './types/MainFilter';
+import { getTodos, addTodo, deleteTodo } from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
+import { Loader } from './components/Loader/Loader';
 
-const USER_ID = 0;
+const USER_ID = 9975;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorOfUpdate, setErrorOfUpdate] = useState('');
+  const [isTodosLoading, setIsTodosLoading] = useState(true);
+  const [filterOption, setFilterOption] = useState(MainFilter.All);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [inputStatus, setInputStatus] = useState(false);
```

Comment:
I'd remove this state and calculate it based on other state values, e.g.:
const isInputDisabled = !!tempTodo && !!loadingIds.length;

Reply:
done